q2studio/api/workspace.py

```diff
@@ -75,8 +76,12 @@ def get_artifacts():
 @fail_gracefully
 def create_artifact():
     request_body = request.get_json()
+    _format = None
+    if request_body['source_format']:
+        _format = request_body['source_format']
```

Comment:
Ok.  I'm not completely sure that we are able to rely on it to be null, but I agree that we should be able to.  I'll check if it works as it should already and if it doesn't I'll follow the code back and figure out what's happening.